jinad/api/endpoints/logs.py

```diff
@@ -1,20 +1,22 @@
-import uuid
 import time
-import json
 import asyncio
 from pathlib import Path
-from typing import Optional
+from typing import Any
+from urllib.parse import parse_qsl
 
 from jina.logging import JinaLogger
-from fastapi import APIRouter, WebSocket, WebSocketDisconnect
+from fastapi import APIRouter, WebSocket
+from starlette.endpoints import WebSocketEndpoint
```

Comment:
maybe it will need to be moved to the inner part where is needed.

This can be done later when is merged in the core

Reply:
yes. This needs handling when moved to core.